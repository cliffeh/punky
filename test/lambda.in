((lambda (x) (* x x)) 7)
