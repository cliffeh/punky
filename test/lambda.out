49
14
144
